README.md

```diff
@@ -1,58 +1,64 @@
-# Breadcrumb
-Breadcrumb is a canonical component that any VTEX app can use.
+# VTEX Breadcrumb
+
+## Description
+The VTEX BreadCrumb is an app that shows a hierarchy of the current page in relation to the store structure and it is used by the Dreamstore product.
+
+:loudspeaker: **Disclaimer:** Don't fork this project, use, contribute, or open issue with your feature request.
 
 ## Release schedule
 | Release  | Status              | Initial Release | Maintenance LTS Start | End-of-life | Dreamstore Compatibility
 | :--:     | :---:               |  :---:          | :---:                 | :---:       | :---: 
 | [0.x]    | **Maintenance LTS** |  2018-05-29     | 2018-11-20            | March 2019  | 1.x
 | [1.x]    | **Current Release** |  2018-11-20     |                       |             | 2.x
 
+See our [LTS policy](https://github.com/vtex-apps/awesome-io#lts-policy) for more information.
+
+## Table of Contents
+- [Usage](#usage)
+  - [Blocks API](#blocks-api)
+    - [Configuration](#configuration)
+  - [Styles API](#styles-api)
+- [Troubleshooting](#troubleshooting)
+- [Tests](#tests)
+
+## Usage
+
+This app uses our store builder with the blocks architecture. To know more about Store Builder [click here.](https://help.vtex.com/en/tutorial/understanding-storebuilder-and-stylesbuilder#structuring-and-configuring-our-store-with-object-object)
 
-To use it in your code you should first add it as a dependencie into your app's ```manifest.json```, like:
+To use this app, you need to add it in your `dependencies` in the `manifest.json` file.
 
 ```json
-"dependencies": {
-    "vtex.breadcrumb": "0.x",
-}
+  dependencies: {
+    "vtex.breadcrumb": "0.x"
+  }
 ```
 
-## Usage
-You can use it in your code through an extension point in your app.
+Then, add the `breadcrumb` block into our app theme, as we do in our [Dreamstore app](https://github.com/vtex-apps/dreamstore/blob/master/store/blocks.json). 
```

Comment:
`breadcrumb` it's not directly imported in the Dreamstore. Maybe `product-details` or `search-result` is a better example.